### PR TITLE
Add example to docs of custom element

### PR DESCRIPTION
**Pull request by bsl**
Related issue: #185

### CODE REVIEW

File: docs/content/basic-syntax.md [hunk 1]
```diff
@@ -79,6 +79,22 @@ Before version 0.18, Maud allowed the curly braces to be omitted. This syntax wa
 
 [#137]: https://github.com/lambda-fairy/maud/pull/137
 
+## Non-standard HTML ("custom") elements
+
+Maud is not restricted to writing well-known HTML elements like `h1`, `p`, `span`, etc. If you are using a framework such as [React][react] or [Vue][vue] that requires the use of [custom elements][custom-elements], it's straightforward.
```
Comment (lambda-fairy) [inline docs/content/basic-syntax.md:84]: Are React and Vue really relevant here? My impression is that they use their own component system -- they can integrate with the Custom Elements API but that's not what most people use them for.

Also, I advise against using "straightforward" (or similar words like "easy" or "trivial") here. Those terms are subjective and don't belong in a technical document.

File: docs/content/basic-syntax.md [hunk 2]
```diff
@@ -79,6 +79,22 @@ Before version 0.18, Maud allowed the curly braces to be omitted. This syntax wa
...
+
+```rust
+html! {
+  blog-post {
+    title { "My blog" }
+  }
+}
```
[the remark this answers is not in the document]
Comment (bsl) [reply docs/content/basic-syntax.md:91]: Oops, I should have noticed that.

File: docs/content/basic-syntax.md (hunk 1 above)
Comment (bsl) [reply docs/content/basic-syntax.md]: I don't know. I've removed the references to specific frameworks.

I have reservations about any form of language policing, but out of gratitude for your work, and keeping the principle of charity in mind, and in the spirit of cooperation and so forth, I've changed it to language that's hopefully innocuous.

**Comment by lambda-fairy** [comment]
Thanks!

**Pull request merged**